src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  };
+
+  static died(animal) {
```

Comment:
think of better name for this method.
All the function names should describe us what they're doing. usually it they start with `get...`, `set...` `calculate...` and so on